shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -180,6 +192,10 @@ void AndroidExternalViewEmbedder::SubmitFrame(
       frame->Submit();
     }
   }
+
+  if (vsync_id) {
```

Comment:
same here. Probably better to have a named constant like: INVALID_VSYNC_ID

Reply:
Done